### PR TITLE
[List View] Show Results Counter

### PR DESCRIPTION
**Pull request by akshaynthakur**
## Summary
This PR adds a counter on the search screen that displays the number of results based on the current search as "X result(s)".

## Test Plan
To test, go to the search screen and try different search values to see the counter in action. 

## Notes
Lines 6-9: outdated import not being used anymore, was causing issues. Was unsure about deletion, so just commented out.

Syntax issue on Line 76-79: had to disable prettier because it doesn't allow {' '} as a valid way to add a space to the text, but a one-line solution doesn't work either under jsx style guidelines.  

### Next Steps
May need to adjust `marginTop` or add `marginBottom` as more components are added to the search screen.

## Screenshots and Tests

https://user-images.githubusercontent.com/78326649/156950559-9f406a52-8ecc-49f6-bcfc-0db43f5d557b.mp4

### CODE REVIEW

File: src/screens/SearchScreen.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ import React, { useEffect, useState } from 'react';
 import { func, shape } from 'prop-types';
 import { ScrollView, Text } from 'react-native';
 import { Searchbar } from 'react-native-paper';
+// import {
```
Comment (jjiang17) [inline src/screens/SearchScreen.js:6]: you'll want to delete commented-out code

**Review by daniellenwong — APPROVED** [review]
This looks good!

File: src/screens/SearchScreen.js [hunk 2]
```diff
@@ -59,6 +63,20 @@ export default function SearchScreen({ navigation }) {
...
+        <Text
+          style={{
+            fontSize: 11,
+            color: '#3B3F51',
+            marginTop: 20,
+            textAlign: 'right',
+            marginRight: 35,
+          }}
+        >
+          {/* eslint-disable-next-line prettier/prettier */}
```
Comment (jjiang17) [inline src/screens/SearchScreen.js:75]: try using f-string concatenation to fix this formatting issue!

**Review by jjiang17 — CHANGES_REQUESTED** [review]
other than the minor prettier issue everything looks good to me! good work 😃😃😃👍🏽👍🏽👍🏽👍🏽👍🏽👍🏽

**Review by jjiang17 — APPROVED** [review]

**Pull request merged**